src/helpers/math.test.js

```diff
@@ -0,0 +1,33 @@
+const math = require('./math');
+
+test('Rounding up', () => {
+    expect(math.roundMoney(1.005)).toBe(1.01);
+});
+
+test('Rounding down', () => {
+    expect(math.roundMoney(1.0049)).toBe(1.0);
+});
+
+// While 0 and -0 work the same in most cases, we want to avoid displaying -0 since the concept of -0 can be confusing for some people
+test('Rounding to -0', () => {
+    var roundedNegZero = math.roundMoney(-0);
+    expect(Object.is(roundedNegZero, 0)).toBe(true);
+});
+
+// This is a more of a pedantic than practical case since values representing money should not reach anywhere close to Number.MAX_VALUE
+// However, rounding Number.MAX_VALUE to two decimal places should not cause it to become infinity
+/*
+test('Rounding a very large number', () => {
+    expect(math.roundMoney(Number.MAX_VALUE)).toBeCloseTo(Number.MAX_VALUE);
+});
+*/
+
+test('Rounding infinity', () => {
+    expect(math.roundMoney(Infinity)).toBe(Infinity);
+});
+
+test('Rounding NaN', () => {
```

Comment:
This is actually a good question to ask - do we want rounding money to return NaN? A lot of times it could be convenient because some values start out as `undefined` and returning `0` instead of `NaN` could be a useful shortcut for getting monetary values. On the other hand it could easily fail silently if an actual issue came up. Would it be useful to return 0 if `undefined` and `NaN` if the value is a non-number in some other way?

Maybe it'd be more useful to find an alternative solution that handles that. I just know it's been a bug-prone annoyance that I've been seeing a lot and it would be great to have a uniform solution.